coursebuilder/common/utils.py

```diff
@@ -339,3 +340,22 @@ def find_youtube_video_id(text):
         if match:
             return match.group(1)
     return None
+
+
+def embed_carousel(content):
+    start_key = '[start_carousel]'
+    stop_key = '[stop_carousel]'
+    check = True
+
+    while check:
+        start = content.find(start_key)
+        end = content.find(stop_key, start)
+        if start != -1 and end != -1:
+            to_be_formatted = content[start + len(start_key):end].split('[split]')
+            to_be_formatted = '<div class="owl-carousel owl-theme course-carousel"><div class="item">' + '</div><div class="item">'.join(
```

Comment:
@andela-angene It could be just me but is this https://github.com/andela-angene/coursebuilder-core/blob/7e629f6bac182e1157e142d3664bfe0b7369c788/coursebuilder/common/utils.py#L355-L358 a bit hard to read?